services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -1160,6 +1183,25 @@ public Builder bearingsList(@Nullable List<Bearing> bearings) {
       return this;
     }
 
+    /**
+     * A radius around a starting point where the routes' provider tries to avoid any
+     * significant maneuvers. Use this option when the vehicle is traveling at a significant
+     * speed to avoid dangerous maneuvers when re-routing. If a route is not found using
+     * the specified value, it will be ignored. Note that if a large radius is used, the routers'
+     * provider may ignore an important turn and return a long straight path before the first
+     * maneuver.
+     * <p>
+     * Note: the param cannot be used with {@link #departAt(String)} and {@link #arriveBy(String)},
+     * since these modes are used for reference requests, not for real-time routing.
+     *
+     * @param avoidManeuverRadius avoid maneuver radius, in meters. Possible range is [0, 1000].
+     * @return this builder for chaining options together
+     */
+    @NonNull
+    public abstract Builder avoidManeuverRadius(
+      @Nullable @IntRange(from = 0, to = 1000) Integer avoidManeuverRadius
```

Comment:
Let's not specify a range here and let server validate it instead. This allows us more flexibility in case the range is expanded.

Reply:
If we provide more than `1000`, is an error returned or is it clamped to `1000`?

Reply:
server returns error 
```
{
  "message": "should be <= 1000",
  "code": "InvalidInput"
}
```

Reply:
we should mention 1000 meters in the docs anyway, so it will be limited anyway

Reply:
Okay, that's fine and matches all other APIs, let's not restrict the range in `mapbox-java` and let server validate.